phpunit.xml

```diff
@@ -16,7 +16,6 @@
     <filter>
         <whitelist processUncoveredFilesFromWhitelist="true">
             <directory suffix=".php">./lib/</directory>
-            <directory suffix=".php">./modules/consent/lib/</directory>
```

Comment:
This line actually broke the PHPunit coverage tool, probably because the module isn't enabled by default. I have removed it, since we're in the process of externalizing the consent-module